src/Minsk/CodeAnalysis/Binding/BoundCompoundAssignmentExpression.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Minsk.CodeAnalysis.Symbols;
+
+namespace Minsk.CodeAnalysis.Binding
+{
+    internal sealed class BoundCompoundAssignmentExpression : BoundExpression
```

Comment:
Having a dedicated bound nodes make perfect sense, keep this.

Reply:
Great!